comp110-worksheet-6/OxoBoard.cs

```diff
@@ -14,13 +14,14 @@ public class OxoBoard
 		// Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
 		public OxoBoard(/* int width = 3, int height = 3, int inARow = 3 */)
 		{
-			throw new NotImplementedException("TODO: implement this function and then remove this exception");
+            string[,] Board = new string[2, 2];
+            return Board;
```

Comment:
You can't return a value from a constructor -- you would instead need to make your array a member variable of your class